wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -230,23 +239,69 @@ Manche Wörter haben Varianten. Ihre bevorzugte Schreibweise ist wie folgt:
 - `erstellt` anstatt `beatmapped` oder `mapped`.
 - `BN` oder `Beatmap Nominators`, wenn auf die *Beatmap Nominators* verwiesen wird.
 
-### Wettbewerbsartikel
+### Turnierartikel
 
 Diese Worte müssen wie folgt verwendet werden (der Abstand muss übereinstimmen):
 
 - `NoMods`, `NoMod` oder `NM`
 - `FreeMods`, `FreeMod` oder `FM`
 - `Tiebreaker`
 
-## Abkürzungen, Akronyme und Initialwörter
-
-Abkürzungen, Akronyme und Initialwörter müssen bei ihrem ersten Auftreten vollständig ausgeschrieben werden. Nachfolgende Vorkommen sind optional, aber werden nur ausgeschrieben, sofern erforderlich. Das Folgende ist ein Beispiel:
+### Übersetzungen für häufige Begriffe
+
+Dieser Abschnitt listet häufig verwendete Begriffe und ihre entsprechenden Übersetzungen auf. Beim Übersetzen von Artikeln sollte darauf geachtet werden, möglichst die hier aufgelisteten Übersetzungen zu verwenden. Ausnahmen können gemacht werden, wenn die ursprüngliche Bedeutung durch die Übersetzung verloren geht.
+
+#### Gameplay
+
+| Englisch | Deutsch |
+| :-: | :-- |
+| `game modifier (mod)` | `Spielmodifikation (Mod)` |
+| `gameplay` | `Spielverlauf` |
+| `multiplier` | `Multiplikator` |
+| `pass rate` | `Erfolgsquote` |
```

Comment:
Where would you use this in articles? Also, at least on the website it's "Success rate" nowadays and it's currently translated as "Erfolgsrate", but feel free to change that in [Crowdin](https://crowdin.com/translate/osu-web/all/en-de?filter=basic&value=0#q=Success%20rate).

```suggestion
```

Reply:
`Client\Interface` still uses "pass rate" and thus I use "Erfolgsquote" there. "Erfolgsrate" is also fine. I can change it accordingly here and in #6988 .

Reply:
After a search, I found that `Beatmap\Category` is the only article that uses "success rate". Honestly, I thought this is used more frequently.

Reply:
Should just be changed in that article then. Makes sense to use the terms used on the website imo

Reply:
```suggestion
| `success rate` | `Erfolgsrate` |
```